### PR TITLE
Extract verify-origami-json from obt

### PR DESCRIPTION
**Pull request by emortong**
Resolves https://github.com/Financial-Times/origami/issues/427

- Extracts verify-origami-json from obt
   - runs it from lint.bash with $workspace argument
- Extracts the `files` helpers and creates an `origami-tools-helpers` module to be used outside of obt

### CODE REVIEW

File: tools/verify-origami-json/index.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+// "service": An HTTP service that follows the service specification
+// "cli": A command line tool
+// "library": A library that is not a front-end component
+// "website": Origami websites that aren’t intended to be services
+// "config": Projects that are configuration for other projects
+// "example": Example and boilerplate projects
+// "meta": Repository-only projects that relate to how Origami works
+// null: An Origami project that does not fit any of the named categories
+
+const cwd = process.argv[2]
```
Comment (chee) [inline tools/verify-origami-json/index.js:22]:
```suggestion
const cwd = process.argv[2] || process.cwd()
```

let's fall back to the current directory so people can run it like `npm exec verify-origami-json` from inside a component

**Review by chee — APPROVED** [review]
i left one comment, but this looks great !!

**Comment by emortong** [comment]
@chee thanks! I had just forgotten to delete the file for verify-origami-json inside obt and the tests, but I have pushed that in the last commit

**Comment by chee** [comment]
@emortong that last failing test is that it wants you to run `npm run regenerate` from the root, and commit any changed files.

**Review by chee — APPROVED** [review]
HONK

**Pull request merged**